plugins/modules/database/misc/redis_set.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Andreas Botzner <andreas at botzner dot com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: redis_set
```

Comment:
Have you thought about renaming this module to `redis_data`, to make clear it belongs to `redis_data_info`? After all it can not just set values (`state=present`), but also delete values (`state=absent`).